plugins/modules/aci_interface_policy_link_level.py

```diff
@@ -35,10 +35,11 @@
     default: true
   speed:
     description:
-    - Determines the interface policy administrative port speed.
+    - Determines the interface policy administrative port speed. 
```

Comment:
```suggestion
    - Determines the interface policy administrative port speed.
```
Please remove the trailing whitespace to help with sanity tests.